pkg/logs/client/http/destination.go

```diff
@@ -223,15 +223,17 @@ func (d *Destination) sendAndRetry(payload *message.Payload, output chan *messag
 
 		d.retryLock.Lock()
 		backoffDuration := d.backoff.GetBackoffDuration(d.nbErrors)
+		d.retryLock.Unlock()
 		d.blockedUntil = time.Now().Add(backoffDuration)
 		if d.blockedUntil.After(time.Now()) {
+			d.retryLock.Lock()
 			log.Warnf("%s: sleeping until %v before retrying. Backoff duration %s due to %d errors", d.url, d.blockedUntil, backoffDuration.String(), d.nbErrors)
```

Comment:
Will be printed on every concurrent send when the EP is not available? Previously only the first send entering sleep would print this (the rest waiting on a lock).

Reply:
Indeed but in the past we had the first send that would not reset the counter after success